### DISCUSSION TITLE
Welcome to printnotes Discussions!

### DISCUSSION THREAD

**Discussion by RoBoT095** [RoBoT095/printnotes#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wick3dr0se** [comment]
@RoBoT095 thanks for making this cool app! Options are limited on Android, at least ones that meet my expectations

I didn't want to start up issues like feature request without discussing the potential of a few things.. Like currently it seems to have basic markdown support but GH flavored markdown would be an improvement maybe (for me it is lol). If it could render mermaid diagrams, that would be sick. Speaking of rendering, it is a bit odd that h3 (`###`) is rendered with an underline. I haven't seen that elsewhere but everything else seems good so far from my testing

Some things I really appreciate, like the render previews, all notes view and directory/file structure. I hope these things don't change. I do think the big header saying 'All Notes' is a bit *extra* but the app otherwise looks beautiful to me. It's minimal, easy to use and actually saves my notes without problems

**Reply by RoBoT095** [reply]
The markdown rendering stuff is all mainly from a library, I only added a few things like syntax for underlining, highlighting, and tags, as well as included examples by the author for HTML syntax rendering and latex. 

I will be honest that it is not the easiest library to use, but flutter doesn't have many good options for markdown libraries, as those that exist are either outdated or are extremely basic with little ways to expand their features. I am only left with writing my own library which is not something I am equipped as this is literally the second app I make with flutter/dart (first one being very basic, Apple app store rejected it for being 'design spam' aka too similar to other stuff), so I am learning as I go.

Furthermore, I did make this app due to wanting something that didn't exist, this is technically a hobby project, but I don't plan to abandon it any time soon as I use it daily myself.
